### PR TITLE
Updated input generation script 

### PR DESCRIPTION
**Pull request by anastasia-shch**
- added config template to be populated with newly generated data and used by the integration tests
- fixed get_chip_from_anno.sh to sync batch names in the prepared basic info and chip table (db)
- updated integration tests

### CODE REVIEW

**Review by RigbeGW — APPROVED** [review]
I guess this is removing the previous commit that added a logic to skip the step if the files existed and as far as i can tell looks ok

File: scripts/README.md
Comment (RigbeGW) [inline scripts/README.md:1]: looks good

File: scripts/prepare_genotypeB_input.sh
Comment (RigbeGW) [inline scripts/prepare_genotypeB_input.sh:1]: I guess this is undoing the previous commit of the logic to skip files if they exist, looks ok to me

File: server/gt_test.py
Comment (RigbeGW) [inline server/gt_test.py:1]: I didnt test this but am sure you have, but one question I have regarding this is the try block starting at line 84 is handling different assertions (not just one) and I guess would be difficult to tell which part is raising the exception so may be would be better to refactor?

**Comment by anastasia-shch** [comment]
@RigbeGW I squashed all commits into one to cleanup history and refactored integration tests as recommended.

**Review by RigbeGW — APPROVED** [review]

**Pull request merged**